superset/databases/commands/update.py

```diff
@@ -75,12 +75,21 @@ def validate(self) -> None:
         if not self._model:
             raise DatabaseNotFoundError()
         database_name: Optional[str] = self._properties.get("database_name")
+        configuration_method: Optional[str] = self._properties.get(
```

Comment:
is this optional? it looks required in the validations below

Reply:
but it should be optional in the update right? THey aren't forced to update it.

Reply:
is it possible for a configuration to have none? Doesn't it get a configuration method in the post method? So it should still have a config method during the update, even if they didn't update the config method specifically. 

Am I understanding the update correctly?

Reply:
we update on the second step on save as well, so all we have is the post response to go by, and so we don't pass the configuration method with it.  it's not possible for the row to have a none value, but since we don't currently update that value, it's possible for the update request not to pass the configuration method.

Reply:
That makes sense, thank you! I changed this to reflect that.